src/Asm89/Stack/Cors.php

```diff
@@ -45,18 +45,10 @@ public function __construct(HttpKernelInterface $app, array $options = array())
 
     public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQUEST, $catch = true)
     {
-        if (!$this->cors->isCorsRequest($request)) {
-            return $this->app->handle($request, $type, $catch);
-        }
-
         if ($this->cors->isPreflightRequest($request)) {
```

Comment:
This is also accessing the `Origin` header on **every** request. The implementation should remove the call to `CorsService::isCorsRequest()`.

Reply:
for cacheability, we should also use the * values when possible then probably.